### ISSUE TITLE
holding c or l breaks the game

### ISSUE THREAD

**Issue by mgula** [mgula/2D#1]
but just pressing them is fine

**Comment by mgula** [comment]
"breaks the game" = disable other keys (except arrow keys) and cause response to arrow keys to be sluggish. Cannot figure this out.

I've also discovered it's not just c or l - but also e, u, a, s ... seems random.

**Comment by mgula** [comment]
scrapped key bindings and tested with key listeners - same issue occurs with the same keys. Arrow keys are fine. But holding certain keys ceases all responsiveness. some console output:

key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=37,keyText=←,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x25] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=37,keyText=←,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x25] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=37,keyText=←,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x25] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=37,keyText=←,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x25] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=37,keyText=←,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x25] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=37,keyText=←,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x25] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=37,keyText=←,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x25] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=37,keyText=←,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x25] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=37,keyText=←,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x25] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=37,keyText=←,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x25] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=37,keyText=←,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x25] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=37,keyText=←,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x25] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=39,keyText=→,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x27] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=39,keyText=→,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x27] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=39,keyText=→,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x27] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=39,keyText=→,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x27] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=39,keyText=→,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x27] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=39,keyText=→,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x27] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=39,keyText=→,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x27] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=39,keyText=→,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x27] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=39,keyText=→,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x27] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=39,keyText=→,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x27] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=39,keyText=→,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x27] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=39,keyText=→,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x27] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=39,keyText=→,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x27] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=38,keyText=↑,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x26] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=38,keyText=↑,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x26] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=38,keyText=↑,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x26] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=38,keyText=↑,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x26] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=38,keyText=↑,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x26] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=38,keyText=↑,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x26] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=38,keyText=↑,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x26] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=38,keyText=↑,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x26] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=38,keyText=↑,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x26] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=38,keyText=↑,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x26] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=38,keyText=↑,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x26] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=38,keyText=↑,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x26] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=40,keyText=↓,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x28] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=40,keyText=↓,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x28] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=40,keyText=↓,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x28] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=40,keyText=↓,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x28] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=40,keyText=↓,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x28] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=40,keyText=↓,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x28] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=40,keyText=↓,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x28] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=40,keyText=↓,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x28] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=40,keyText=↓,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x28] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=40,keyText=↓,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x28] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=40,keyText=↓,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x28] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=40,keyText=↓,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x28] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=40,keyText=↓,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x28] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=40,keyText=↓,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x28] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=40,keyText=↓,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x28] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=40,keyText=↓,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x28] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=40,keyText=↓,keyChar=Undefined keyChar,keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x28] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]
key pressed. details:java.awt.event.KeyEvent[KEY_PRESSED,keyCode=67,keyText=C,keyChar='c',keyLocation=KEY_LOCATION_STANDARD,rawCode=0,primaryLevelUnicode=0,scancode=0,extendedKeyCode=0x43] on views.Game1View[,0,0,1280x479,layout=java.awt.FlowLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=9,maximumSize=,minimumSize=,preferredSize=]

tl;dr: all arrow keys are good, then I hold c and it breaks. after that all keys other than arrow keys don't respond. can really only just scratch my head at this one

**Comment by mgula** [comment]
Had a thought - c and l break the keyboard, so what if I just don't use those keys? ... well, I removed all functionality tied to the c key, and holding it still causes the issue (should have known already since other unimplemented keys do the same thing)
